src/foam/u2/crunch/CapabilityInterceptView.js

```diff
@@ -41,50 +41,78 @@ foam.CLASS({
   ],
 
   messages: [
-    { name: 'TITLE', message: 'Welcome to Capability unlock options' },
-    { name: 'CAP_TITLE', message: 'Capabilities Available' },
-    { name: 'SUBTITLE_1', message: 'You do not have access to undertake your previous selected action.' },
-    { name: 'SUBTITLE_2', message: 'Please select one of the following capabilities, to unlock full feature.' }
+    { name: 'TITLE', message: 'Permission Required' },
```

Comment:
@kgrgreer please comment on renaming Capabilities to Permissions.   This does not fit all scenarios.

Reply:
We intercept permissions and then find a capability that grants it, correct? So, it is always a capability that kicks this off, isn't it?

Reply:
Yes, we want to retain the idea of 'Capability' in the system.  @kennyqiyenkan Logan we need to rework this language.

Reply:
I am thinking "Hold On - You are currently incapable of performing this action. Please refer to the following list on how to obtain it." It's a bit more agnostic, still implies it is an interrupt, and provides enough information. I think. I ran this past Logan. Will change when he gets back to me if good.

Reply:
Sounds good.

Reply:
Changed it to what I mentioned above with the exception of the title. It's now "Missing Capability"